### PR TITLE
Fix ipcamera command format

### PR DESCRIPTION
**Pull request by bvanwijk**
This fixes the formatting of the IPCamera commands.
I have also remove the author notation in every class.

### CODE REVIEW

**Comment by NaomidR** [comment]
Good processing of the feedback. The changes look good, also no warnings so I agree with merging.

File: backend/src/main/java/com/benine/backend/camera/ipcameracontrol/IPCamera.java [hunk 1]
```diff
@@ -402,13 +405,13 @@ public String toJSON() throws CameraConnectionException {
...
     json.put("id", this.getId());
     try {
-      json.put("pan", getPosition().getPan());
-      json.put("tilt", getPosition().getTilt());
-      json.put("zoom", (double) getZoomPosition());
-      json.put("focus", (double) getFocusPosition());
-      json.put("autofocus", isAutoFocusOn());
-      json.put("iris", (double) getIrisPosition());
-      json.put("autoiris", isAutoIrisOn());
+      json.put("pan", new Double(getPosition().getPan()));
```
Comment (jochem725) [inline backend/src/main/java/com/benine/backend/camera/ipcameracontrol/IPCamera.java]: Why is it required to create a new object from an already existing double object?

Comment (bvanwijk) [inline backend/src/main/java/com/benine/backend/camera/ipcameracontrol/IPCamera.java]: I've no idea, I think I choose the wrong version due to a merge conflict.

**Comment by jochem725** [comment]
I think this can be merged.

**Pull request merged**